pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm"]
+requires = ["setuptools>=67.0.0", "setuptools-scm>=7.1.0"]
```

Comment:
Any specific reason to use v67. pyproject.toml is supported by setuptools starting with v61.

Reply:
Not entirely specific but I went through https://setuptools.pypa.io/en/latest/history.html and version v66.1.0

Reply:
Ok it make sens then.